Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.ts

```diff
@@ -86,6 +95,31 @@ export class UserDetailComponent implements OnInit, OnDestroy {
     );
   }
 
+  public openChangePopup(): void {
+    const wrapper = document.getElementsByClassName('pb-margin-20')[0];
```

Comment:
There is a config for this `backdropClass`: https://material.angular.io/components/dialog/api#MatDialogConfig

But please let us discuss beforehand if this should be implemented app-wide or not.

Reply:
@Zeroks77